schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
 Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
+framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established a framework agreement. The suppliers on the framework are listed in the award section of the related process.
 planning,Planning process,This contracting process follows on from the related planning process.
 parent,Parent contract (for sub-contracts),This contracting process might result in a sub-contract of the related process.
-prior,Prior process,This contracting process is the renewal or replacement of the related prior process.
-unsuccessfulProcess,Unsuccessful process,This contracting process follows on from an previous unsuccessful process.
-subContract,Sub-contract,The related process might result in a sub-contract of this contract.
-replacementProcess,Replacement process,The related process might result in the replacement of this contract.
-renewalProcess,Renewal process,The related process might result in the renewal of this contract.
+prior,Prior process,This contracting process follows on from the related successful process.
+prior.replacement,Prior process (replacement),This contracting process is the replacement of the related process.
+prior.renewal,Prior process (renewal),This contracting process is the renewal of the related process.
+prior.additional,Prior process (additional),"This contracting process is for the purchase of additional goods, services or works related to the goods, services or works purchased from the same supplier in the related process."
+prior.repetition,Prior process (repetition),"This contracting process is for the purchase of new goods, services or works that constitute a repetition of the goods, services or works purchased from the same supplier in the related process."
```

Comment:
```suggestion
prior.repetition,Prior process (repetition),"The contracting process aims to the purchase new goods, services or works constituting a repetition of the goods, services or works purchased from the same supplier in the related contracting process."
```

Reply:
Done in b17fab75bd496a9e980003a7e38cbd68c5e1bb62 (to avoid committing a typo)